04-http-lifecycle/02-routing.adoc

```diff
@@ -422,6 +422,18 @@ Route
   .middleware(['auth'])
 ----
 
+==== namespace
+Often we have multiple controllers grouped inside the same folder. With the namespace directive you can specify the folder on the groupe.
+
+[source, js]
```

Comment:
Can you also show the folder structure here, to make it clear, which part should be inside the `namespace` function

Reply:
I just updated the documentation with the example from the PR.